packages/vite/types/package.json

```diff
@@ -0,0 +1,3 @@
+{
+  "//": "this file is just here to make pnpm happy with --frozen-lockfile"
```

Comment:
@zkochan this directory is linked using `link:` protocol and I had to add this empty package.json here because `pnpm` throws when installing with `--frozen-lockfile` (or on CI):

```
 ERR_PNPM_NO_IMPORTER_MANIFEST_FOUND  No package.json (or package.yaml, or package.json5) was found in "/Users/evan/Vue/vite-pnpm/packages/vite/types".
```

Normal install without `--frozen-lockfile` is fine though.

Reply:
I see. I will look into it. If it works without `--frozen-lockfile`, then it should work with it as well.

Reply:
This workaround is not needed anymore with pnpm [v5.16.2](https://github.com/pnpm/pnpm/releases/tag/v6.15.2)